### PR TITLE
Stress

### PR DESCRIPTION
**Pull request by samwaseda**
This was related to the fact that `extract_compute` (as well as `extract_atoms`) was **not** ordered according to the structure definition, i.e. it has to be reordered using `id` with `extract_atoms`.

### CODE REVIEW

File: pyiron/lammps/interactive.py [hunk 1]
```diff
@@ -563,23 +563,16 @@ def interactive_stress_getter(self):
...
-                        j + (j != k) * (k + 2)
-                    ]
-                    for i in range(len(self.structure))
-                    for j in range(3)
-                    for k in range(3)
-                ]
-            ).reshape(len(self.structure), 3, 3)
-            / 1.602e6
-        )
+        ID = self._interactive_library.extract_atom("id", 0)
```
Comment (jan-janssen) [inline pyiron/lammps/interactive.py]: `id` is a python function and writing it in capital letters is even less pythonic - can not we use something like `id_lst` or `atom_ids` ?

File: pyiron/lammps/interactive.py [hunk 2]
```diff
@@ -563,23 +563,16 @@ def interactive_stress_getter(self):
...
+        ID = np.array([ID[i] for i in range(len(self.structure))])-1
+        ID = np.arange(len(ID))[np.argsort(ID)]
+        ind = np.array([0, 3, 4, 3, 1, 5, 4, 5, 2])
+        ss = self._interactive_library.extract_compute("st", 1, 2)
+        ss = np.array([ss[i][j] for i in range(len(self.structure)) for j in range(6)]).reshape(-1, 6)[ID]
+        ss = ss[:, ind].reshape(len(self.structure), 3, 3)/1.602e6
```
Comment (jan-janssen) [inline pyiron/lammps/interactive.py]: Please do not use numbers like `1.602e6 `- but instead define the contents using `scipy.constants`

Comment (jan-janssen) [reply pyiron/lammps/interactive.py]: That seems to be the wrong constant: 
```
from scipy import constants
constants.eV*constants.bar*constants.angstrom**3
``` 
equals `1.6021766208000003e-44`

Comment (samwaseda) [reply pyiron/lammps/interactive.py]: `1/constants.eV*constants.bar*constants.angstrom**3`

Comment (jan-janssen) [reply pyiron/lammps/interactive.py]: Previously you had `1/1.602e6` now it is `1/1.6021766208000003e-44` - so it seem like there is a factor missing.

Comment (samwaseda) [reply pyiron/lammps/interactive.py]: `1/constants.eV*constants.bar*constants.angstrom**3 = 6.241509125883258e-07`

Comment (samwaseda) [reply pyiron/lammps/interactive.py]: `1/1.602e6 = 6.242197253433208e-07`

Comment (jan-janssen) [reply pyiron/lammps/interactive.py]: It is still not the same like `1.602e6` which you had previously.

Comment (jan-janssen) [reply pyiron/lammps/interactive.py]: Now I get it. Looks fine to me.

**Pull request merged**